cookbooks/aws-parallelcluster-config/files/default/cloudwatch_agent/cloudwatch_log_files.json

```diff
@@ -265,6 +265,84 @@
         }
       ]
     },
+    {
+      "timestamp_format_key": "default",
+      "file_path": "/var/log/sssd/sssd.log",
+      "log_stream_name": "sssd",
+      "schedulers": [
+        "slurm",
+        "plugin"
+      ],
+      "platforms": [
+        "centos",
+        "ubuntu",
+        "amazon"
+      ],
+      "node_roles": [
+        "HeadNode"
+      ],
+      "feature_conditions": [
+        {
+          "dna_key": [
+            "directory_service",
+            "enabled"
+          ],
+          "satisfying_values": ["true"]
+        }
+      ]
+    },
+    {
+      "timestamp_format_key": "default",
+      "file_path": "/var/log/sssd/sssd_default.log",
+      "log_stream_name": "sssd_default",
```

Comment:
default is the name of the domain. can we maybe call this `sssd_default_domain`? or just `sssd_domain`?

Reply:
Done